packages/vanilla/src/helpers/js/_core.js

```diff
@@ -330,6 +333,14 @@ class Core {
     return this._defaults;
   }
 
+  setTheme(theme) {
+    Themes.checkTheme(theme);
+    this.themedElements.forEach((el) => {
```

Comment:
👌, I was struggling to figure out how I would implement this and this is a decent way to be explicit.